hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -509,6 +509,256 @@ def check_core_dumps(reason=''):
     return str(hard_core_dump_value)
 
 
+def check_service_status(service_name, state):
+    '''
+    Ensure that the given service is not running on the system. Return False if the service is running.
+    Return True otherwise
+    state can be ebaled or disabled.
+    '''
+    output = _execute_shell_command('systemctl is-enabled ' + service_name + ' >/dev/null 2>&1; echo $?')
+    if (state == "disabled" and output.strip() == "1") or (state == "enabled" and output.strip() == "0"):
+        return True
+    else:
+        return _execute_shell_command('systemctl is-enabled ' + service_name + ' 2>/dev/null')
+
+def check_ssh_timeout_config(reason=''):
+    '''
+    Ensure SSH Idle Timeout Interval is configured
+    '''
+
+    client_alive_interval = _execute_shell_command("grep \"^ClientAliveInterval\" /etc/ssh/sshd_config | awk '{print $NF}'").strip()
+    if client_alive_interval != '' and int(client_alive_interval) < 300:
+    	client_alive_count_max = _execute_shell_command("grep \"^ClientAliveCountMax\" /etc/ssh/sshd_config | awk '{print $NF}'").strip()
+    	if client_alive_count_max != '' and int(client_alive_count_max) < 3:
+       	    return True
+    	else:
+            return "ClientAliveCountMax value should be less than 3"
+    else:
+    	return "ClientAliveInterval value should be less than 300"
+
+
+def check_unowned_files(reason=''):
+    '''
+    Ensure no unowned files or directories exist
+    '''
+
+    unowned_files = _execute_shell_command("df --local -P | awk 'NR!=1 {print $6}' | xargs -I '{}' find '{}' -xdev -nouser 2>/dev/null").strip()
```

Comment:
Can you please create new user and create few files using that user and try this check again or both local and mounted files ?